client/js/editmode.js

```diff
@@ -1419,7 +1419,24 @@ function populateAddWidgetOverlay() {
       toAdd.z = getMaxZ(dice3D.get('layer')) + 1;
       toAdd.faces = Array.from({length: sides}, (_, i) => sides == 6 ? i + 1 : ""+(i+1));
       toAdd.shape3d = result.d2 ? 'd2-flip' : true;
-
+      if (sides == 6){
```

Comment:
Should we apply this in dice.css rather than just widgets that get added?

Reply:
Yes.